README.md

```diff
@@ -555,26 +557,47 @@ These are the steps for manual set-up of certificates:
    $ KUBCONFIG="<<your cluster kubeconfig path>> PATH="$PATH:$PWD/_work/bin" ./test/setup-ca-kubernetes.sh
 ```
 
-- **Deploy the driver to Kubernetes using DeviceMode:LVM**
+- **Deploy the driver to Kubernetes**
 
-```sh
-    $ sed -e 's/PMEM_REGISTRY/<your registry>/' deploy/kubernetes-<kubernetes version>/pmem-csi-lvm.yaml | kubectl create -f -
-```
+The `deploy/kubernetes-<kubernetes version>` directory contains
+`pmem-csi*.yaml` files which can be used to deploy the driver on that
+Kubernetes version. The files in the directory with the highest
+Kubernetes version might also work for more recent Kubernetes
+releases.
+
+For each Kubernetes version, four different deployment variants are provided:
+
+   - `direct` or `lvm`: one uses DeviceMode:Direct, the other DeviceMode:LVM
```

Comment:
Shouldn't DeviceMode:Direct and DeviceMode:LVM be within quotes? 
i.e.: `DeviceMode:Direct`

Reply:
And yes, a final bullet is missing to this first bullet.

Reply:
Added.

Is it okay now?

@okartau if @marcemq doesn't get around to giving her okay, please go ahead and merge. We need to get the number of pending PRs down if we want to release this week.

Reply:
Yes, it is ok now.